lib/parameter_parser/base_parameter_parser.rb

```diff
@@ -8,6 +8,10 @@ class ParseError < StandardError
     def initialize(errors)
       @errors = errors
     end
+
+    def error
+      errors.join(". ")
```

Comment:
The API only returns a single string, and I didn't want to change that.

The unit tests for the parser are also (annoyingly) all testing for a string, via a copy of this method on
`search_parameter_parser`. I initially tried moving it but decided it wasn't worth the time to change the tests.